src/Mammon/Services/CostCentreReportService.cs

```diff
@@ -6,6 +6,7 @@ public class CostCentreReportService (IConfiguration configuration, CostCentreRu
 	private string BlobStorageContainerName => configuration[Consts.DotFlyerAttachmentsContainerNameConfigKey]!;
 	private IEnumerable<string> EmailToAddresses => configuration[Consts.ReportToAddressesConfigKey]!.SplitEmailContacts();
 	private string EmailFromAddress => configuration[Consts.ReportFromAddressConfigKey]!;
+	private int ReportBillingPeriodStartDayInMonth => int.Parse(configuration[Consts.ReportBillingPeriodStartDayInMonthConfigKey]!);
```

Comment:
should we do a try parse here and use a default value if something goes wrong? what's the risk here?

Reply:
the config is validated separately... see CostCentreReportServiceConfigValidator